src/test/unit/timer_definition_unittest.include/drivers/timer.h

```diff
@@ -0,0 +1,21 @@
+#include <mock_enums.h>
```

Comment:
For consistency and maintainability, these stubs should all be included in the main test `.cc` file.

Reply:
OK. #6759 has it.